lib/llm/OpenAIClient.ts

```diff
@@ -138,10 +147,47 @@ export class OpenAIClient extends LLMClient {
       },
     });
 
-    const response = await this.client.chat.completions.create({
+    delete openAiOptions.requestId;
+    delete openAiOptions.image;
+
+    const formattedMessages: ChatCompletionMessageParam[] =
+      options.messages.map((message) => {
+        if (Array.isArray(message.content)) {
+          const contentParts = message.content.map((content) => {
+            if ("image_url" in content) {
+              return {
+                image_url: {
+                  url: content.image_url.url,
+                },
+                type: "image_url",
+              } as ChatCompletionContentPartImage;
```

Comment:
assign to variable of type `ChatCompletionContentPartImage` and return/reference that variable instead of `as`